radio/src/telemetry/frsky.cpp

```diff
@@ -69,7 +69,7 @@ FrskyData frskyData;
 
 #if defined(CPUARM)
 uint8_t telemetryProtocol = 255;
-#define IS_FRSKY_D_PROTOCOL()      (telemetryProtocol == PROTOCOL_FRSKY_D)
+#define IS_FRSKY_D_PROTOCOL()      (telemetryProtocol == PROTOCOL_FRSKY_D || telemetryProtocol == PROTOCOL_FRSKY_D_INVERTED)
```

Comment:
This breaks on all ARM platforms but sky9x (the reason Travis check failed) because you defined `PROTOCOL_FRSKY_D_INVERTED` only for REVX. Same at line 616 below